misc/config_definitions.json

```diff
@@ -4771,6 +4771,13 @@
             "section": "prometheus",
             "order": 3
         },
+	"prometheus.prefix": {
+	    "default": "",
+	    "type": "text",
+            "group": "poller",
```

Comment:
Indentation is off and you'll need to add strings to resources/lang/en/settings.php

Reply:
I believe I fixed the indentation in the subsequent commit.  Looking at settings.php now thanks :)

Reply:
OK thanks for the assistance with this, I believe the latest commit also adds the required to resources/lang/en/settings.php